packages/edit-site/src/components/navigation-sidebar/navigation-panel/main-back-button.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalUseSlot as useSlot,
+	createSlotFill,
+} from '@wordpress/components';
+
+const slotName = '__experimentalNavigationPanelMainBackButton';
+
+const { Fill, Slot: NavigationPanelMainBackButtonSlot } = createSlotFill(
```

Comment:
I'm wondering whether we should create a new slot for this or reuse the existing one from interface package. 🤔
https://github.com/WordPress/gutenberg/blob/65258a077e4899ee4933c79077108f56f8b5ce6d/docs/designers-developers/developers/slotfills/main-dashboard-button.md

While the W button was the same in post and site editor we opted for reusing it. In any case this documentation should be updated for site editor case, because the icon example there will no longer work.

Reply:
I may be misunderstanding, but the slot here is for the "< Dashboard" button inside the panel, and not for the main "W" button itself.

Reply:
I was thinking about resuing `MainDashboardButton` too. It would make sense but for some reason, I have a feeling to create a new slot. That just sounds better for me.

Reply:
> I may be misunderstanding, but the slot here is for the "< Dashboard" button inside the panel and not for the main "W" button itself.

Yes, but they serve the same purpose - overriding the button that takes users to main dashboard. We don't need a slot for overriding the main "W" button anymore since there is no point in doing that, and it would break sidebar functionality.

> I was thinking about resuing MainDashboardButton too. It would make sense but for some reason, I have a feeling to create a new slot. That just sounds better for me.

My preference would be to reuse it. It's practically the same code and the existing name is still semantically accurate, and in my opinion more descriptive than this one. Also it's not out of the question that "back to dashboard" button will be moved out of navigation in future designs, since some concerns were already raised about its discoverability. If it gets moved somewhere else, tying the slot's name to `NavigationPanel` would be too specific and we'd need to change it, which becomes a problem if the exported component is no longer experimental.